django-backend/fecfiler/web_services/dot_fec/test_dot_fec_serializer.py

```diff
@@ -14,6 +14,7 @@ class DotFECSerializerTestCase(TestCase):
         "test_committee_accounts",
```

Comment:
Not sure where to put this comment. You'll need to update `compose_report_level_memos` in the dot_fec_composer to only return report memos and update the transactions section to insert transaction memos after their transactions